packages/plugin-core/src/utils/common/getStageFromPkgJson.ts

```diff
@@ -0,0 +1,17 @@
+import { Stage } from "@dendronhq/common-all";
+
+/**
+ * Extrapolates the stage from the 'name' property in the package.json manifest.
+ * For node environments, getStage() is sufficient, but in environments that
+ * aren't node based (where process.env is unavailable), this method suffices.
+ * NOTE: This method does not return 'test'
+ */
+export function getStageFromPkgJson(packageJSON: any): Stage {
```

Comment:
Agree that webpack is probably the more correct thing to do, but it's also a lot of additional work + difficult testing.  We would need a separate webpack config for development, nightly, and production + modify all the build scripts accordingly.  

I also don't think examining the package manifest is the _worst_ methodology - can we defer webpack for now?  I'll convert this to a private method within web logic so it's not used elsewhere.

Reply:
So `process.env.STAGE` will be set correctly whenever webpack runs at compile time? Can you point me to where we set that environment variable?

Reply:
it will be if we set it. we currently don't but webpack automatically sets NODE_ENV. https://webpack.js.org/configuration/mode/

we also inspect `NODE_ENV` in src/env.ts for when calling `getStage`

Reply:
as discussed offline, going to try `getStage`